Formula/organize-tool.rb

```diff
@@ -0,0 +1,173 @@
+class OrganizeTool < Formula
+  include Language::Python::Virtualenv
+
+  desc "Organize - The file management automation tool"
+  homepage "https://github.com/tfeldmann/organize"
+  url "https://files.pythonhosted.org/packages/0f/3e/41166d703a3c571583e5200c40deae8c831f597e37ce2df21d9274a80d39/organize-tool-1.9.1.tar.gz"
+  sha256 "368a265f66f2d5cb2f01e6c76a3d98cac1b1ff6a82c2f5d534f3abb5ca95ad44"
+  license "MIT"
+
+  bottle :unneeded
```

Comment:
```suggestion
```
Just remove this, as a bottle is needed for this formula. BrewTestBot will add the `bottle` block upon merging.

Please squash all your changes into cone commit, and use the commit message
```
organize-tool 1.9.1 (new formula)
```
as specified in the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md#to-add-a-new-formula-for-foo-version-234-from-url).